### ISSUE TITLE
Regex literals being incorrectly formatted

### ISSUE THREAD

**Issue by guillaumealgis** [nicklockwood/SwiftFormat#1333]
Similar to https://github.com/nicklockwood/SwiftFormat/issues/1319, reproduced in 0.50.6 :  

```swift
let match = try! /FOOBAR: (\d{15})/
```

gets formatted as 

```swift
let match = try! /FOOBAR: (\d { 15 })/
```

**Comment by nicklockwood** [comment]
Thanks for reporting this, I've pushed a fix to the develop branch.

The trigger for the issue is  the `try`, so removing that would be a workaround for now, or you could put `// swiftformat:disable:next all` on the previous line.

**Comment by guillaumealgis** [comment]
Great! Thank you @nicklockwood for the quick answer and fix!

**Comment by nicklockwood** [comment]
Fixed in 0.50.7
